dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/trackedentity/TrackedEntityAttributeStoreTest.java

```diff
@@ -113,6 +113,18 @@ public void setUpTest()
                 attributeService.getTrackedEntityAttributeByName( "Attribute" + s ) ) )
             .collect( Collectors.toList() );
 
+        // Setting searchable to true for 5 tracked entity type attributes
+        TrackedEntityTypeAttribute teta = teatList.get( 0 );
```

Comment:
Is this setup only relevant for the tests you added? Does it make tests run slower? If not, all good. If so, please move it only into the relevant tests 😄

Reply:
It does not make the tests slower. As all the attributes were already part of the test setup. I only changed a few parameters before it gets persistetd. So the setup time should be same.